ui/app/AppLayouts/Chat/ChatColumn/ChatMessages.qml

```diff
@@ -12,341 +12,395 @@ import "../../../../imports"
-            authorCurrentMsg: msgDelegate.ListView.section
-            // The previous message is actually the nextSection since we reversed the list order
-            authorPrevMsg: msgDelegate.ListView.nextSection
-            imageClick: imagePopup.openPopup.bind(imagePopup)
-            messageId: model.messageId
-            emojiReactions: model.emojiReactions
-            linkUrls: model.linkUrls
-            communityId: model.communityId
-            hasMention: model.hasMention
-            stickerPackId: model.stickerPackId
-            pinnedMessage: model.isPinned
-            pinnedBy: model.pinnedBy
-            gapFrom: model.gapFrom
-            gapTo: model.gapTo
-
-            // This is used in order to have access to the previous message and determine the timestamp
-            // we can't rely on the index because the sequence of messages is not ordered on the nim side
-            prevMessageIndex: {
                 // This is used in order to have access to the previous message and determine the timestamp
                 // we can't rely on the index because the sequence of messages is not ordered on the nim side
-                if (msgDelegate.DelegateModel.itemsIndex < messageListDelegate.items.count - 1) {
-                    return messageListDelegate.items.get(msgDelegate.DelegateModel.itemsIndex + 1).model.index
+                prevMessageIndex: {
+                    // This is used in order to have access to the previous message and determine the timestamp
+                    // we can't rely on the index because the sequence of messages is not ordered on the nim side
+                    if (msgDelegate.DelegateModel.itemsIndex < messageListDelegate.items.count - 1) {
+                        return messageListDelegate.items.get(msgDelegate.DelegateModel.itemsIndex + 1).model.index
+                    }
+                    return -1;
                 }
-                return -1;
+                nextMessageIndex: {
+                    if (msgDelegate.DelegateModel.itemsIndex <= 1) {
+                        return -1
+                    }
+                    return messageListDelegate.items.get(msgDelegate.DelegateModel.itemsIndex - 1).model.index
+                }
+                scrollToBottom: chatLogView.scrollToBottom
+                timeout: model.timeout
             }
-            nextMessageIndex: {
-                if (msgDelegate.DelegateModel.itemsIndex <= 1) {
-                    return -1
+        }
+    }
+
+
+    Rectangle {
+        id: userList
+        visible: showUsers && chatsModel.channelView.activeChannel.chatType !== Constants.chatTypeOneToOne
+
+        property int defaultWidth: 250
+
+        SplitView.preferredWidth: visible ? defaultWidth : 0
+        SplitView.minimumWidth: 50
+        
+        color: Style.current.secondaryMenuBackground
+
+        anchors.top: parent.top
+        anchors.bottom: parent.bottom
+
+        height: childrenRect.height
```

Comment:
This height and the  anchors above conflict. You'll need to keep only one of the other